mlflow/models/cli.py

```diff
@@ -155,9 +155,10 @@ def prepare_env(
 @commands.command("build-docker")
 @cli_args.MODEL_URI
 @click.option("--name", "-n", default="mlflow-pyfunc-servable", help="Name to use for built image")
+@cli_args.ENV_MANAGER
 @cli_args.INSTALL_MLFLOW
```

Comment:
Can we update the docstring for `INSTALL_MLFLOW` to indicate that MLflow will be installed into the virtualenv as well? Right now, it just mentions conda:

```
  --install-mlflow     If specified and there is a conda
                       environment to be activated mlflow will
                       be installed into the environment after
                       it has been activated. The version of
                       installed mlflow will be the same asthe
                       one used to invoke this command.
```

Reply:
Updated!